ProactivePolicyTemplate/build.sh

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env bash
+
+##########################################################################
```

Comment:
Copy pasted without changes from https://github.com/cake-build/example/blob/master/build.sh

Reply:
(will come back to this)